### PR TITLE
Migrate community e2e tests

### PR DESCRIPTION
**Pull request by RedbackThomson**
Migrating E2E tests from the `community` repository. See parent issue - https://github.com/aws-controllers-k8s/community/issues/735

Changes:
- Added references to `resources_directory` and `bootstrap_directory` in `__init__.py`
- Update all references to `load_resource_file` to now use `load_apigatewayv2_resource` (to reference `bootstrap_directory`)
- Modified `service_bootstrap/cleanup.py` files with new `__main__` entrypoints

### CODE REVIEW

**Comment by RedbackThomson** [comment]
Please don't merge this for the time being - it still has references to temporary paths and is subject to further breaking changes.

**Comment by RedbackThomson** [comment]
Ping @aws-controllers-k8s/elasticache-maintainer This is ready to review and merge, please :)

**Review by jaypipes — APPROVED** [review]

**Pull request merged**
